src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -2,7 +2,7 @@
   <v-form id="correct-nr-form" ref="correctNrForm" v-model="formValid" lazy-validation>
     <v-row no-gutters>
       <v-col cols="1" class="mt-1">
-        <v-btn x-small fab outlined :ripple="false" color="#495057" class="step-icon">1</v-btn>
+        <v-btn x-small fab outlined :ripple="false" color="gray7" class="step-icon" tabindex="-1">1</v-btn>
```

Comment:
I added `tabIndex="-1"` to remove these from the tab order (in other words, pressing tab skips over them). This is nice for accessibility AND for tabbing between NR number and phone/email.

Reply:
Cool trick, taking note of this one!

Reply:
Is this something that can be applied to generic html elements or is this a vuetify component prop specifically?

Reply:
HTML

Reply:
very cool, ty